app/models/game.rb

```diff
@@ -36,6 +36,17 @@ def check?
     kings_in_check?('white') || kings_in_check?('black')
   end
 
+  def forfeit(current_user)
+    if current_user.id == white_player_id
+      white_player_id.update_attributes(white_player_id.losses: white_player_id.losses + 1)
+      black_player_id.update_attributes(black_player_id.wins: black_player_id.wins + 1)
+    else
+      white_player_id.update_attributes(white_player_id.wins: white_player_id.wins + 1)
+      black_player_id.update_attributes(black_player_id.losses: black_player_id.losses + 1)
+    end
+    white_player_id, black_player_id = nil, nil # end the game and make unavailable
```

Comment:
Would it be similar to add winning_player_id IS NULL to the scope so that when the winning_player is set, the game is no longer available? or is it better to create a game status and update it?